### PR TITLE
Fix/misc

### PR DESCRIPTION
**Pull request by mariannemagnussen**

### CODE REVIEW

File: src/molecules/QuickSearch/QuickSearch.pcss [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
   .telia-textfield-content {
     border-radius: 100px;
-    border: 1px solid var(--grey-200);
+    border: 1px solid var(--grey-200) !important;
```
Comment (HeleneGrini) [inline src/molecules/QuickSearch/QuickSearch.pcss:20]: hm. Do we need important or can we be more specific with the selectors?

Comment (mariannemagnussen) [reply src/molecules/QuickSearch/QuickSearch.pcss:20]: Yes, thats actually possible, thanks!

File: src/molecules/Dropdown/Dropdown.pcss [hunk 2]
```diff
@@ -105,6 +105,17 @@
...
   }
+
+  &__disabled {
+    pointer-events: none;
+    background-color: var(--grey-100);
+    color: var(--grey-400);
+    border-color: var(--grey-300);
+  }
+
+  &__outline.telia-dropdown-toggle__purple {
```
Comment (HeleneGrini) [inline src/molecules/Dropdown/Dropdown.pcss:116]: should this also have `telia-dropdown-toggle__disabled` ?

Comment (mariannemagnussen) [reply src/molecules/Dropdown/Dropdown.pcss]: good catch

**Review by HeleneGrini — APPROVED** [review]

**Pull request merged**